### ISSUE TITLE
How to avoid output saturation?

### ISSUE THREAD

**Issue by zaitoichi** [xanderchf/MonoDepth-FPN-PyTorch#15]
I'm training your model on the NYU Depth V2 dataset but can't avoid the model's predictions quickly degenerating to outputting a depth value for the entire image. How did you avoid this? The model doesn't appear to learn at all, the loss doesn't decrease, it just oscillates around the same value, and very little gradient is passed to any of the parameters, if any. I've tried different learning rates with the same result.
